apps/project/tests/test_apis.py

```diff
@@ -55,6 +61,9 @@ def test_create_project(self):
         data = {
             'title': 'Test project',
             'data': {'testKey': 'testValue'},
+            'organizations': [
+                {'organization': Organization.objects.all()[0].id, 'type': ProjectOrganization.DONOR},
```

Comment:
Mostly test is atomic in nature. So everything you create will be revert back after a test is done.
Try to create object on the same scope if possible. Basically move `self.org` inside `test_create_project` and use it directly instead of `Organization.objects.all()[0]`. This also avoids confusion on how there is an organization. Thanks.